vue/components/organisms/ripe-image/ripe-image.vue

```diff
@@ -113,6 +113,357 @@ export const RipeImage = {
+            type: Boolean,
+            default: null
+        },
+        /**
+         * Displays the Debug information box.
+         */
+        initialsDebug: {
+            type: Boolean,
+            default: null
+        },
+        /**
+         * Overrides the initialsProfile's font to be applied on the initials.
+         */
+        fontFamily: {
+            type: String,
+            default: null
+        },
+        /**
+         * Overrides the initialsProfile's font weight to be applied on the initials.
+         */
+        fontWeight: {
+            type: String,
+            default: null
+        },
+        /**
+         * Overrides the initialsProfile's font size to be applied on the initials.
+         */
+        fontSize: {
+            type: String,
+            default: null
+        },
+        /**
+         * Overrides the initialsProfile's spacing between each letter.
+         */
+        fontSpacing: {
+            type: Number,
+            default: null
+        },
+        /**
+         * Font Trim.
+         */
+        fontTrim: {
+            type: Number,
+            default: null
+        },
+        /**
+         * Mask strategy when using raster fonts: 'self' means that the alpha channel of the
+         * letter image is going to be used to defined both which pixels are going to be passed
+         * in the paste operation and the intensity; 'simple' means that just the pixels with
+         * a valid alpha value (greater than zero) will be passed to the target image.
+         */
+        fontMask: {
+            type: String,
+            default: null
+        },
+        /**
+         * Forces a specific font mode, may improve text render (vector fonts) - it's used by
+         * some graphics drivers to indicate what mode the driver prefers; usually when the
+         * font uses antialiasing the mode 'L' shall improve rendering.
+         */
+        fontMode: {
+            type: String,
+            default: null
+        },
+        /**
+         * Line Height.
+         */
+        lineHeight: {
+            type: Number,
+            default: null
+        },
+        /**
+         * Line break, is optional and can have one of (normal and word_break).
+         */
+        lineBreaking: {
+            type: Boolean,
+            default: null
+        },
+        /**
+         * Overrides the initialsProfile's shadow, which defines if the initials have a shadow.
+         */
+        shadow: {
+            type: Boolean,
+            default: null
+        },
+        /**
+         * Overrides the initialsProfile's color of the shadow to be used.
+         */
+        shadowColor: {
+            type: String,
+            default: null
+        },
+        /**
+         * Overrides the initialsProfile's offset to be applied on the shadow.
+         */
+        shadowOffset: {
+            type: String,
+            default: null
+        },
+        /**
+         * Offsets. Example:

```

Comment:
Offset of what?

Reply:
I guess that if this was copied from somewhere we should update it everywhere with a proper explanation :/

Reply:
Yeah, kinda thought the same, but as it was like this in https://github.com/ripe-tech/plugin-base/blob/master/templates/config/applier.html.tpl I let it like that

Reply:
Go for the 2 for the price of 1 fix 💪